### ISSUE TITLE
HelloRadio: Voice Command not working :(

### ISSUE THREAD

**Issue by Edwin-Camargo** [EdgeTX/edgetx-user-manual#47]
I recently bought a radio from you because of its features, such as being able to “talk” to it to issue commands.

My radio arrived this past Saturday, but my joy was short-lived: it responded normally when I said “HelloRadio”, but shortly afterwards it stopped responding. Maybe I changed some settings by mistake, but I think I did what I was supposed to do.

The radio is an ELRS version, Mode 2 (LEFT (the label is wrong)) and is configured as follows:
a) Speech recognition Management: Voice = LUA and Power Switch = ON (enabled);
b) CH56 Release: I tried both ways: on and off
c) I have already updated the radio’s firmware with the version available on your website (V16-firmware-2.10.1-092401-EM.bin) and it didn’t work.
d) I have already deleted a model I created (a model aircraft I created for testing) and it didn’t work either.

Attached some images of how the configuration works.

I don't know what else to do. Can recording another microSD card solve the problem?

Could someone help?

The radio no longer “talks” to me and this is one of the main features that led me to buy it. I'm very upset ☹

Thank you for your help.

Att.
![Firmware version](https://github.com/user-attachments/assets/fc6a4040-27b9-44c7-b137-d51aea4063c3)
![IMG_20241201_201609](https://github.com/user-attachments/assets/8b6c16f3-396e-4e73-b58f-9e2e4d0e16e2)
![IMG_20241202_211311](https://github.com/user-attachments/assets/48371f10-1a11-4dfb-8f64-47b91b54b9d7)
![Set Voice LUA on](https://github.com/user-attachments/assets/7a7e7796-ad57-40c0-a3ed-f61542cf30f4)

Edwin Camargo

**Comment by helloradiosky** [comment]
hi, our after-sales person have contacted you already, is your problem solved?

**Comment by helloradiosky** [comment]
Hello Edwin, thank you very much for your support. Checked with the after-sales person, the reason is that the re-seller sent you the wrong version. We are very sorry for this. If you have any other issues, we will solve them for you. We will urge the distributor to be more careful with their job to avoid similar problems from happening again. Thank you again for your support.

**Comment by Edwin-Camargo** [comment]
> Hello Edwin, thank you very much for your support. Checked with the after-sales person, the reason is that the re-seller sent you the wrong version. We are very sorry for this. If you have any other issues, we will solve them for you. We will urge the distributor to be more careful with their job to avoid similar problems from happening again. Thank you again for your support.

Hello!

The problem was solved by Ken (HelloRadio Support), but the problem was not because the AliExpress seller sent the wrong product (I bought the 4in1 version and he sent the ELRS), but because of something unknown: the original sound files were corrupted, but Ken helped with the solution.

As for the AliExpress seller (RC Car Mall), after I informed him that the wrong version came, he insisted on saying that "only the label on the box was wrong, but the radio sent was the one I bought", anyway, this kind of attitude is a shame...

As for Support (Ken), congratulations!

Best regards,

Edwin
